mangaki/mangaki/management/commands/dedupe.py

```diff
@@ -0,0 +1,113 @@
+from django.core.management.base import BaseCommand, CommandError
+
+import os
+import csv
+from dedupe import Dedupe, consoleLabel, canonicalize
+from mangaki.models import Work
+
+
+def dedupe_training(category):
+    assert (category in ['mangas', 'animes']),"Only mangas or animes needs training"
+    data = {}
+    for work in Work.objects.filter(category__slug=category[:len(category)-1]):
```

Comment:
It seems you're only using `title` and `vo_title`, so let's add a `values_list` after the `filter` to make the query faster.
I'd also recommend to query the category first, then use the category ID rather than the slug.